Source/AliveLibAO/Elum.cpp

```diff
@@ -3718,7 +3717,14 @@ void Elum::VUpdate_4102A0()
             }
             
             (this->*sElum_motion_table_4C5148[field_FC_current_motion])();
-            
+
+            if (oldMotion != field_FC_current_motion &&
```

Comment:
can prob delete this debug code if its no longer needed?

Reply:
still needed the 3 motions are still unknown